lib/pry/method/weird_method_locator.rb

```diff
@@ -98,7 +107,9 @@ def pry_file?
       #   superclass method.
       def find_method_in_superclass
         guess = method
-
+        if skip_superclass_search?
+          return guess
```

Comment:
Nice, but the purpose of this class is to return the correct method for the binding, does this achieve this?

Reply:
It returns a Pry::Method for the prepended module:
```
[1] pry(#<Pry::Method::WeirdMethodLocator>)> p guess
=> #<Pry::Method:0x007ff2090a99e0
 @method=#<Method: Foo(AlsoPrependThis)#foo>,
 @source="def foo\n  super + 2\nend\n",
 @visibility=nil>
```

For the Hanami case this does not seem to matter.